### PR TITLE
Update UI, add search feature

### PR DESCRIPTION
**Pull request by dndquynh**
add + search demo
delete_demo

### CODE REVIEW

**Comment by hieupham7127** [comment]
looks good but still missing minor features before merge.

- new tab after clicking on a search entry
- sync/local selector
- sorted list from search

**Comment by hieupham7127** [comment]
please also include opened issues (or open them if necessary)

**Comment by dndquynh** [comment]
* new tab after clicking on a search entry :white_check_mark:
 * sync/local selector
 * sorted list from search :white_check_mark:

File: popup/popup.html [hunk 1]
```diff
@@ -6,46 +6,64 @@
 
         <title>Go Links</title>
 
+        <link href="/../dist/all.min.css" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (dndquynh) [reply popup/popup.html:9]: It's for Font Awesome icons.

**Comment by dndquynh** [comment]
Fixed #4 #9.
#3 #6 will be implemented in a different PR.

File: popup/popup.js [hunk 2]
```diff
@@ -37,3 +41,89 @@ saveButton.onclick = () => {
...
+$(document).ready(function(){
+    $(searchQuery).on("keyup", function() {
+      var value = $(this).val().toLowerCase();
+      $("#item-list li").filter(function() {
+        $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
+      });
+    });
+});
+
+$("#add").on("click", function() {
```
Comment (hieupham7127) [inline popup/popup.js:121]: `#add` is a bit vague for naming tags. Please use a more specific name.

File: popup/popup.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+}
+.button-outline:hover {
+    background-color: #3bad74;
+    color: white;
+}
+
+/* Help section */
+#help {
+    display: none;
+}
```
Comment (hieupham7127) [inline popup/popup.css:145]: newline

File: popup/popup.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    border: none;
+    border-bottom: solid 1px lightgrey;
+    border-radius: 0;
+}
+.search-box:focus {
+    border: none !important;
+    border-bottom: solid 1px #3bad74 !important;
+}
+
+#icon {
```
Comment (hieupham7127) [inline popup/popup.css:52]: avoid naming ids as simpletons like `icon`, `url`, etc., it's confusing and they should be reserved for other purposes.

File: popup/popup.js [hunk 5]
```diff
@@ -2,13 +2,13 @@ const golinkInput = document.getElementById("go-link")
...
 const saveButton = document.getElementById("save")
 const searchQuery = document.getElementById("search")
-const mainDiv = document.getElementById("main")
-const helpDiv = document.getElementById("help")
-const helpButton = document.getElementById("help-btn")
-const homeButton = document.getElementById("home-btn")
+const mainDiv = document.getElementById("main-section")
+const helpDiv = document.getElementById("help-section")
+const helpButton = document.getElementById("btn-help")
+const homeButton = document.getElementById("btn-home")
```
Comment (hieupham7127) [inline popup/popup.js:8]: inconsistent naming (should be `helpBtn` and `homeBtn` instead)

File: popup/popup.js [hunk 6]
```diff
@@ -2,13 +2,13 @@ const golinkInput = document.getElementById("go-link")
...
 const itemList = document.getElementById("item-list")
 const addSection = document.getElementById("add-section");
```
Comment (hieupham7127) [inline popup/popup.js:10]: you haven't adjusted the name for this one. I assume it should be `section-expand`?

File: popup/popup.js [hunk 7]
```diff
@@ -118,12 +118,12 @@ $(document).ready(function(){
...
 
-$("#add").on("click", function() {
+$(expandBtn).on("click", function() {
     addSection.style.display = "block";
     this.style.display ="none";
 });
 
-$("#minus-button").on("click", function() {
-    addEntry.style.display = "flex";
+$("#btn-collapse").on("click", function() {
```
Comment (hieupham7127) [inline popup/popup.js:126]: you should make this into a variable to be in sync with other tags

**Pull request merged**